cli/cli.go

```diff
@@ -346,8 +346,7 @@ func SetupCLI(args []string) error {
 		&cli.StringFlag{
 			Name:    "log-modules",
 			Aliases: []string{"m"},
-			Usage: "`LIST` of logging modules (levels) to " +
-				"include in the output.",
+			Usage: "Warning: log-modules is deprecated.",
```

Comment:
Maybe "warning" is a bit strong since this is only the help screen. I would also add for clarity that it has no effect.

Reply:
So, something along the lines of:
`Note: log-modules have no effect`?

Reply:
Maybe `-log-modules is accepted for compatibility with old versions but has no effect`